hutch_python/epics_arch.py

```diff
@@ -113,9 +261,30 @@ def print_dry_run(exp_name):
     --------
     >>> epicsarch-qs xpplv6818 --dry-run
     """
-    data = get_questionnaire_data(exp_name)
-    for item in data:
-        print(item)
+
+    qs_data = get_questionnaire_data(exp_name)
+
+    """
+    Updating experiment file.
+    """
+
+    af_path = EPICS_ARCH_FILE_PATH.format(exp_name[0:3]) + 'epicsArch_' + exp_name + '.txt'
+    af_data = read_archfile(af_path)
+    if not os.path.exists(af_path):
+        raise OSError('print_dry_run, invalid path: %s' % af_path)
+    elif os.path.exists(af_path):
+        updated_archFile = check_for_duplicates(qs_data, af_data)
+
+        for item in updated_archFile:
+            print(item)
+
+
+def get_key(val, my_dict):
+    for k, v in my_dict.items():
+        if val == v:
+            return k
+    strError = ""
+    return strError
```

Comment:
Q: What's this blank error message used for?

Reply:
If I am not wrong, an empty string is the same thing as returning false or 0? Might be totally wrong though.

Reply:
Will change to return None